### ISSUE TITLE
Addition of travel history and social event attendance information to import template

### ISSUE THREAD

**Issue by Iheanacho2027** [SORMAS-Foundation/SORMAS-Project#1917]
### Situation Description
Information on social events and travel history is missing on import template

**Comment by bernardsilenou** [comment]
you mentioned in another issue that the import template has much variables, adding travel history will not instead increase it?

**Comment by Iheanacho2027** [comment]
At the moment, travel history is very important as required by the national CIF, missing it out when importing over 200 cases and going back to enter travel history information on a case by case basis is waste of valuable time which would have been saved if they were in the import template. That's why a custom import template as raised in the other ticket would be very important.

**Comment by Iheanacho2027** [comment]
@bernardsilenou any hope on this?.

**Comment by bernardsilenou** [comment]
@Iheanacho2027 The current state of this issue is not ready for implementation, we still need to work on it.
- Please can you add your suggestion in term of mockup before the end of this sprint?
- One challenge i can see is the fact that one case can have many travel histories, how do you intent to handle this in the import. You know one row on the import template is for one case
- Please add a  template of the data you wish to import

**Comment by MateStrysewske** [comment]
@Iheanacho2027 @bernardsilenou Actually the current refactoring of epi data will affect this because we will replace travels, burials and gatherings with exposures (which will still have the relevant fields of the three obsolete entities). Please keep that in mind. Bernard is right though that we need this issue to be specified before we can do anything about it.
